content/pages/how-to/use-direct-upload-with-continuous-integration.md

```diff
@@ -0,0 +1,178 @@
+---
+pcx-content-type: how-to
+title: Use Direct Upload with continuous integration
+---
+ 
+# Use Direct Upload with continuous integration
+ 
+Cloudflare Pages now supports directly uploading prebuilt assets, allowing you to use custom build steps for your applications and deploy to Pages with [Wrangler](/workers/cli-wrangler/).
+ 
+ 
+## Deploying with Wrangler
+ 
+In your project directory, install [Wrangler](/workers/cli-wrangler/) so you can deploy a folder of prebuilt assets by running this command:
+ 
+```sh
+# Publish created project
+CF_ACCOUNT_ID=<account id> npx wrangler pages publish <directory> --project-name=<project name>
+```
+ 
+ 
+## Generate API Token from Cloudflare
+ 
+First log  into the [Cloudflare dashboard](https://dash.cloudflare.com/profile/api-tokens) >  **My Profile** > **API Tokens** > **Create Token** and under "Create Custom Token", select **Get started**. Start by naming the API Token, specify what product you are creating the API token for and give it the edit permission. In this case, select the "Account", "Cloudflare Pages" and "Edit":
+ 
+![Select API token for Cloudflare Pages](../media/select-api-token-for-pages.png)
+ 
+Click on **Continue to summary** and then **Create Token**. Now that you have created this token, you can use it to push your project from continuous integration platforms.
+ 
+## Using GitHub Actions
+ 
+[GitHub Actions](https://docs.github.com/en/actions) is a continuous integration and continuous delivery (CI/CD) platform that allows you to automate your build, test, and deployment pipeline when using GitHub. You can create workflows that build and test every pull request to your repository or deploy merged pull requests to production.
+ 
+After setting up your project initially, you can set up a GitHub Action to automate your subsequent deployments with Wrangler.
+ 
+### Add Cloudflare credentials to GitHub secrets
+ 
+In the GitHub action you have set up, environment variables are needed to push your project up to Cloudflare Pages. Add the values of these environment variables in your project's GitHub repository by going to your projects repository > **Settings** > **Secrets** > **Actions** > **New repository secret**.
+ 
+Add the value of your CLOUDFLARE_ACCOUNT_ID and CLOUDFLARE_API_TOKEN; As `CF_ACCOUNT_ID` and `CF_API_TOKEN`, this will ensure that these secrets are secure, and each time your Action runs, it will access these secrets.
```

Comment:
We should use `CLOUDFLARE_` instead of `CF_` (CF works, but is deprecated)
```suggestion
Add the value of your CLOUDFLARE_ACCOUNT_ID and CLOUDFLARE_API_TOKEN; As `CLOUDFLARE_ACCOUNT_ID` and `CLOUDFLARE_API_TOKEN`, this will ensure that these secrets are secure, and each time your Action runs, it will access these secrets.
```